### PR TITLE
aws-c-s3: add version 0.1.29 with update dependencies

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **aws-c-s3/0.1.29**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by toge** [comment]
It's funny.
I don't think I've committed any SDL changes, but for some reason  CI had detecting them.
I don't know how to resolve this.
I may have to recreate the PR again.

**Comment by madebr** [comment]
I think the c3i bot messed up. sdl received a version bump yesterday.
Try closing and re-opening the pr.

**Review by SSE4 — APPROVED** [review]

File: recipes/aws-c-s3/all/conanfile.py [hunk 1]
```diff
@@ -69,10 +69,13 @@ def package(self):
     def package_info(self):
         self.cpp_info.filenames["cmake_find_package"] = "aws-c-s3"
         self.cpp_info.filenames["cmake_find_package_multi"] = "aws-c-s3"
+        self.cpp_info.set_property("cmake_file_name", "aws-c-s3")
         self.cpp_info.names["cmake_find_package"] = "AWS"
         self.cpp_info.names["cmake_find_package_multi"] = "AWS"
+        self.cpp_info.set_property("cmake_target_name", "AWS")
```
Comment (ericLemanissier) [inline recipes/aws-c-s3/all/conanfile.py:75]:
```suggestion
```

File: recipes/aws-c-s3/all/conanfile.py [hunk 2]
```diff
@@ -69,10 +69,13 @@ def package(self):
...
         self.cpp_info.components["aws-c-s3-lib"].names["cmake_find_package"] = "aws-c-s3"
         self.cpp_info.components["aws-c-s3-lib"].names["cmake_find_package_multi"] = "aws-c-s3"
+        self.cpp_info.components["aws-c-s3-lib"].set_property("cmake_target_name", "aws-c-s3")
```
Comment (ericLemanissier) [inline recipes/aws-c-s3/all/conanfile.py:78]:
```suggestion
        self.cpp_info.components["aws-c-s3-lib"].set_property("cmake_target_name", "AWS::aws-c-s3")
```
I _think_ these targets must be absolute now, right ?

Comment (ericLemanissier) [reply recipes/aws-c-s3/all/conanfile.py:78]: same as on the other PR. this requires conan 1.43.0 which is not yet deployed on CCI, so feel free to ignore these suggestions

**Review by ericLemanissier — APPROVED** [review]

**Pull request merged**